src/main/webapp/js/instructorFeedbackEdit.js

```diff
@@ -652,20 +652,27 @@ function disableRow(elem, row) {
 }
 
 function feedbackRecipientUpdateVisibilityOptions(elem) {
-    elem = $(elem);
-    if (elem.val() === 'OWN_TEAM' || elem.val() === 'TEAMS' ||
-            elem.val() === 'INSTRUCTORS' || elem.val() === 'OWN_TEAM_MEMBERS') {
-        enableRow(elem, 1);
-        disableRow(elem, 3);
+    var $elem = $(elem);
+    if (isRecipientsTeamMembersVisibilityOptionHiddenForRecipientType($elem.val())) {
```

Comment:
Hmm, I don't really think this is the best choice. It's more like checking if "recipient team members" is an invalid option right? That the row is "hidden" is really the effect of this branch.